src/Security/Authentication/OpenIdConnect/src/OpenIdConnectHandler.cs

```diff
@@ -516,6 +552,113 @@ private async Task HandleChallengeAsyncInternal(AuthenticationProperties propert
         throw new NotImplementedException($"An unsupported authentication method has been configured: {Options.AuthenticationMethod}");
     }
 
+    // TODO GetConfigString and ConfigFlagEnabled are only used in PAR, and won't be necessary after
+    // Microsoft.IdentityModel.Protocols.OpenIdConnect is updated to v7.6.1 or later.
```

Comment:
We've updated to 8.0.0, so we should be able to use `PushedAuthorizationRequestEndpoint` and the like now.

Reply:
Awesome, I switched to it!

Reply:
Out of curiosity, how are the dependencies managed? The build system is ... well it's a lot for the uninitiated 😄 

I found eng/BaselineDesigner.props, but it has this:
```
  <ItemGroup Condition=" '$(PackageId)' == 'Microsoft.AspNetCore.Authentication.JwtBearer' AND ('$(TargetFramework)' == '$(DefaultNetCoreTargetFramework)' OR '$(TargetFramework)' == 'net8.0') ">
    <BaselinePackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="[7.0.3, )" />
  </ItemGroup>
```
That looks like 7.0.3 is being referenced, which is clearly not the case.

Reply:
The dependency versions we use come from [Version.props](https://github.com/dotnet/aspnetcore/blob/dfc5189cb5b9ecfd0e0c43310a9355e768bbed45/eng/Versions.props#L14). BaselineDesigner.props is to catch unintentional breaking changes in servicing builds. This should provide some more context.

https://github.com/dotnet/aspnetcore/blob/dfc5189cb5b9ecfd0e0c43310a9355e768bbed45/eng/targets/ResolveReferences.targets#L11-L17